src/MIT-open-group.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+    <license isOsiApproved="false" licenseId="MIT-open-group"
+             name="MIT Open Group variant">
+        <text>
+            <copyrightText>
+                <p>Copyright &lt;yyyy, yyyy&gt;</p>
+            </copyrightText>
+            <p>Permission to use, copy, modify, distribute, and sell this
+                software and its documentation for any purpose is hereby granted
+                without fee, provided that the above copyright notice appear in
+                all copies and that both that copyright notice and this
+                permission notice appear in supporting documentation.
+            </p>
+            <p>The above copyright notice and this permission notice shall be
+                included in all copies or substantial portions of the Software.
+            </p>
+            <p>THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+                EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+                OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+                NONINFRINGEMENT. IN NO EVENT SHALL BE LIABLE FOR ANY CLAIM,
+                DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+                TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+                THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+            </p>
+            <p>Except as contained in this notice, the name of shall not be used
```

Comment:
In this sentence, there are two places where there are words missing from the example submitted by the submitter. You can see that there are missing words between: `the name of ____ shall not be used` and also at the end: `without prior written authorization from ____.`

Looking at an example, for instance at https://gitlab.freedesktop.org/xorg/app/iceauth/-/blob/master/COPYING, you can see that "The Open Group" is used in these locations.

Ideally we don't want to just hard-code "The Open Group" into the template, because then it wouldn't match if somebody else puts their own name there. But we also don't want to leave it empty, because then it will not match under the matching guidelines.

What we can do instead is to insert "The Open Group" but include markup for an alt match. This would mean, in place of the first blank, put the following:

`<alt match=".+" name="name1">The Open Group</alt>`

and in place of the second blank at the end of the sentence, put the following (only the "name" property is changed):

`<alt match=".+" name="name2">The Open Group</alt>`

Reply:
Thank You for taking out time to explain it in a detailed manner.
Having seen `alt` in other licenses, I quite didn't understand the exact use-case.

This cleared my confusion. Thanks!